docs/topics/channels/deploying.txt

```diff
@@ -0,0 +1,196 @@
+Deploying
+=========
+
+Deploying applications using Channels requires a few more steps than a normal
```

Comment:
Do you want to treat "Channels" as a proper noun or not? The capitalization seems inconsistent throughout the docs but maybe I missed some distinction. I think I'd say no given other precedent like "model" and "expression". Happy to make these changes while reviewing the docs but wanted to ask first.